controllers/verticalscalingpolicy_controller.go

```diff
@@ -54,6 +53,7 @@ type VerticalScalingPolicyReconciler struct {
 func (r *VerticalScalingPolicyReconciler) Reconcile(ctxt context.Context, req ctrl.Request) (ctrl.Result, error) {
 	_ = context.Background()
 	_ = r.Log.WithValues("verticalscalingpolicy", req.NamespacedName)
+	var driftedTargets []string
```

Comment:
Could you make this a map to avoid duplicates? Although it's not possible to have duplicates now, we can make this future proofed

Reply:
Sure I will make that change, good call